### PR TITLE
[6_2_X][TIMOB-25359] Android: Fix Ti.UI.ScrollView when keyboard is active

### PR DESCRIPTION
**Pull request by garymathews**
- Cherry-pick of https://github.com/appcelerator/titanium_mobile/pull/9502

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-25359)

### CODE REVIEW

**Comment by lokeshchdhry** [comment]
@garymathews , 
When I build SDK for this PR I get errors:
```
[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java:193: error: cannot find symbol

[javac] 			if (context instanceof TiBaseActivity) {
    [javac] 			                       ^
    [javac]   symbol:   class TiBaseActivity
    [javac]   location: class TiUIScrollView.TiVerticalScrollView

[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java:194: error: cannot find symbol

[javac] 				Window window = ((TiBaseActivity) context).getWindow();
[javac] 				^

[javac]   symbol:   class Window

[javac]   location: class TiUIScrollView.TiVerticalScrollView

[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java:194: error: cannot find symbol

[javac] 				Window window = ((TiBaseActivity) context).getWindow();
    [javac] 				                  ^
    [javac]   symbol:   class TiBaseActivity
[javac]   location: class TiUIScrollView.TiVerticalScrollView

[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java:197: error: package WindowManager does not exist

[javac] 				if ((softInputMode & WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN) == 0) {
    [javac] 				                                  ^

[javac] /Users/lchoudhary/Desktop/repositories/titanium_mobile/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java:198: error: package WindowManager does not exist

[javac] 					window.setSoftInputMode(softInputMode | WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE);
    [javac] 					                                                     ^

[javac] Note: Some input files use or override a deprecated API.

[javac] Note: Recompile with -Xlint:deprecation for details.
[javac] Note: Some input files use unchecked or unsafe operations.

[javac] Note: Recompile with -Xlint:unchecked for details.

[javac] 5 errors

[javac] 1 warning
```

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java [hunk 1]
```diff
@@ -188,6 +188,17 @@ protected int getMeasuredHeight(int maxHeight, int heightSpec)
 		public TiVerticalScrollView(Context context, LayoutArrangement arrangement)
 		{
 			super(context);
+
+			// TIMOB-25359: allow window to re-size when keyboard is shown
```
Comment (lokeshchdhry) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java]: Three imports are missing:
`import org.appcelerator.titanium.TiBaseActivity;`
`import android.view.Window;`
`import android.view.WindowManager;`

**Review by ypbnv — COMMENTED** [review]
Looks good other than the imports pointed by @lokeshchdhry

**Comment by garymathews** [comment]
Updated PR

**Comment by lokeshchdhry** [comment]
FR Passed.

`scrollView` is scrollable when keyboard is up.

Studio Ver:     4.10.0.201709271713
SDK Ver:        6.2.3 local build
OS Ver:         10.12.3
Xcode Ver:      Xcode 8.3.3
Appc NPM:       4.2.9
Appc CLI:       6.2.4
Ti CLI Ver:     5.0.14
Alloy Ver:      1.9.14
Node Ver:       7.10.1
Java Ver:       1.8.0_101
Devices:        ⇨ google Nexus 5 --- Android 6.0.1

**Review by ypbnv — APPROVED** [review]
CR: Pass

**Pull request closed without merging**